apps/aecore/src/aec_tx_sign.erl

```diff
@@ -124,7 +127,7 @@ deserialize([?SIG_TX_TYPE, ?SIG_TX_VSN, TxSer, Sigs]) ->
     #signed_tx{data = Tx, signatures = Sigs}.
 
 %% deterministic canonical serialization.
--spec serialize_to_binary(signed_tx()) -> binary().
+-spec serialize_to_binary(signed_tx()) -> deterministic_signed_tx_binary().
```

Comment:
I propose binary_signed_tx. The type need not include the information that the result of the function is deterministic.

Reply:
ok, adding change now.

Reply:
@ThomasArts Addressed in new commit.